sdk/program/src/bpf_loader_upgradeable.rs

```diff
@@ -119,6 +119,11 @@ impl UpgradeableLoaderState {
     }
 }
 
+/// Returns the program data address for a program ID
+pub fn get_program_data_address(program_address: &Pubkey) -> (Pubkey, u8) {
+    Pubkey::find_program_address(&[program_address.as_ref()], &id())
+}
```

Comment:
Since the bump seed is never used, how about just returning the Pubkey?

Reply:
I can't think of a hypothetical reason why someone downstream might need it. I certainly don't for the migration code. But I feel like someone one day will ask "why don't you guys export the bump too?".

If you'd rather we omit it for now though and only add it when we need it, I can do that!

Reply:
This is the only line I see the bump seed used: https://github.com/solana-labs/solana/blob/28a320d81571f776c439a4090aedcceffaf5fbbb/programs/bpf_loader/src/lib.rs#L576 and everywhere else it's dropped.

Let's omit it for now, and if we decide to create `get_program_data_address_and_bump_seed`, we can do it later